spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink-build/pom.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This pom is not used anymore?

Reply:
Right it's parent for apps which is problematic like this because maven doesn't know about it as a module. We use something like:
```
./mvnw build-helper:parse-version versions:set -DprocessAllModules=true -DgenerateBackupPoms=false -DnewVersion='${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}'
```
to change versions during releases and it doesn't happen with apps.

Reply:
>>Right it's parent for apps which is problematic like this because maven doesn't know about it as a module
This is to avoid duplication in the apps. I can add it as a module `...tasklauncher-sink-dependencies`  to share dependencies, but the build configuration breaks the metadata-plugin unless I duplicate all that in each project.